packages/governance/test/swingsetTests/committeeBinary/bootstrap.js

```diff
@@ -22,8 +22,8 @@ const makeVoterVat = async (log, vats, zoe) => {
 /**
  *
  * @param {Pick<QuestionDetails, 'issue' | 'positions' | 'electionType'>} qDetails
- * @param {TimestampValue} closingTime
- * @param {{ electorateFacet: import('../../../src/committee.js').CommitteeElectorateCreatorFacet, installations: Record<string, Installation>, timer: TimerService }} tools
+ * @param {import('@agoric/time/src/types').Timestamp} closingTime
```

Comment:
note: this replaces `TimestampValue` (which is just an integer) with the preferred `TimeStamp` (which is currently either an integer or a branded record, but the next step of #6003 is to require them all to be branded)